src/ocamlorg_data/tutorial.ml

```diff
@@ -20,6 +21,7 @@ type t =
 let all = 
 [
   { title = {js|Up and Running with OCaml|js}
+  ; fpath = {js|tutorials/en/000_up_and_running.md|js}
```

Comment:
Is `{js|...|js}` a vestige of when the frontend was in ReScript? I think it doesn't have any meaning in standard OCaml?

Reply:
That was indeed introduced for compatibility with ReScript. When trying to remove the `js` part last time, I found that some strings contain the `|}` character, so the strings were escaped unintentionnally.

Keeping the `js` namespace seemed like a good workaround 🙂 

Thanks for the review!

Reply:
It's indeed a bit odd to not just use `".."` here. But not super important and could be done in another PR.